src/pages/public/StagingServerOverview/components/ServerOverview.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faExternalLinkAlt } from "@fortawesome/free-solid-svg-icons";
+
+import { fetchServers, StagingServerAPIResponse } from "./FetchUtils";
+
+import "../../../../App.css";
+
+const ServerOverview = (): JSX.Element => {
+  const [servers, setServers] = useState<[StagingServerAPIResponse]>();
+  const [rows, setRows] = useState<Array<Number>>([]);
```

Comment:
Variable naming somewhat unclear.
Maybe rename to something like `expandedRows` and use `boolean` over `0` and `1`
Something like:
```suggestion
  const [expandedRows, setExpandedRows] = useState<Array<Boolean>>([]);
```